assets/src/application/ui/layout/espressoTable/index.ts

```diff
@@ -6,15 +6,16 @@ import TableRow from './table-row';
 import TableHeaderCell from './table-header-cell';
 import TableDataCell from './table-data-cell';
 
-export const EspressoTable = {
-	Table,
-	TableHeader,
-	TableBody,
-	TableRow,
-	TableFooter,
-	TableHeaderCell,
-	TableDataCell,
-};
+// export const EspressoTable = {
+// 	Table,
+// 	TableHeader,
+// 	TableBody,
+// 	TableRow,
+// 	TableFooter,
+// 	TableHeaderCell,
+// 	TableDataCell,
+// };
```

Comment:
This is a temp change, need to discuss how we want to implement that vertical mode for table.